### DISCUSSION TITLE
Información de abogados

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#77] [Show and tell]
Buenas noches investigue un poco y esto consegui de tipos de demanda, que datos requiere, información y ejemplo de un doc

[Practica semana 1.docx](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/files/8121550/Practica.semana.1.docx)

**Comment by JorgeEHE** [comment]
Muchas gracias por la info @GloriaCoralCerecedo , me ayudara mucho para darme una idea para crear los diagramas y buyer persona al momento de que haga mi práctica!!!

**Comment by GloriaCoralCerecedo** [comment]
Claro, no hay de que 👀👍🏻

**Comment by OscarIslasH** [comment]
Una duda, esto sirve para el buyer persona o para que sirve? Segun acabe de entender hay que hacer el buyer persona de el abogado y el demandante y un wireframe de la pagina

En total era: 
1.- Responder el documento de word 
2.- Hacer ambos buyer persona  
3.- Hacer lo del publico objetivo del cual nos pasaron la plantilla de miro  
4.- Hacer el wireframe usando las paginas y herramientas que se nos paso 

Tenia pensado acabar hoy lo mas rapido posible para poder entregar todo y no atrasarme, si es que voy bien, podrias decirme si no me falto algo? siento que casi no entendi mucho por falta de tiempo, gracias de antemano.

Documento de world: https://github.com/OscarIslasH/Mision-Frontend/blob/main/0.-Reqierimientos.doc
Buyer persona del abogado: https://www.hubspot.es/make-my-persona?persona=-Mwih66SbF6tCgFkHOdx
Publico objetivo: https://miro.com/app/board/uXjVOKNkGxA=/

Aun no acabo lo del wireframe, voy a estar todo el dia haciendolo, por favor si me falta algo diganme.

**Reply by GloriaCoralCerecedo** [reply]
Te faltaría lo del wireframe que mencionas y la UI (interfaz de usuario)

**Reply by JuanSalvador73** [reply]
Tenemos además hasta el 4 de marzo para entregarlo. Pero haces bien en hacerlo ahora. Solo cuida que tengas las asignaciones completas.

**Reply by OscarIslasH** [reply]
@GloriaCoralCerecedo 

Estoy haciendo el Wireframe y ya casi acabo, es este: https://miro.com/app/board/uXjVOJwm_Ic=/?invite_link_id=814440224581 

Lo del UI entiendo se puede hacer en Balsamic. ¿Al final es que si debo pasar todo lo que hice en Miro del Wireframe pasarlo a Balsamic para ver una version funcional?

**Reply by OscarIslasH** [reply]
Creo que tambien se podia en Adobe XD y no se que otra pagina o sofware, si me lo pudieras aclarar, seria todo, muchas gracias por responderme todas mis dudas

**Comment by JuanSalvador73** [comment]
Y gracias por la info. Servirá mucho.

**Comment by ChanLuis** [comment]
Gracias chica
